components/Editor.js

```diff
@@ -146,6 +146,10 @@ class Editor extends React.Component {
     const timestamp = this.state.timestamp ? `_${formatTimestamp()}` : ''
 
     return this.getCarbonImage({ format, type: 'blob' }).then(url => {
+      if (format === 'copy') {
+        window.location = url;
```

Comment:
Is there a way you can open it in a new tab? I think you can do this similar to how we handle the download case, but without setting "download".